protocol-designer/src/components/modals/CreateFileWizard/EquipmentOption.tsx

```diff
@@ -30,45 +33,60 @@ export function EquipmentOption(props: EquipmentOptionProps): JSX.Element {
     disabled = false,
     ...styleProps
   } = props
+
+  const [targetProps, tooltipProps] = useHoverTooltip()
+
   return (
-    <Flex
-      aria-label={`EquipmentOption_flex_${text}`}
-      alignItems={ALIGN_CENTER}
-      width="21.75rem"
-      padding={SPACING.spacing8}
-      border={
-        isSelected && !disabled ? BORDERS.activeLineBorder : BORDERS.lineBorder
-      }
-      borderRadius={BORDERS.borderRadiusSize2}
-      cursor={disabled ? 'auto' : 'pointer'}
-      backgroundColor={disabled ? COLORS.darkGreyDisabled : COLORS.transparent}
-      onClick={disabled ? undefined : onClick}
-      {...styleProps}
-    >
-      {showCheckbox ? (
-        <Icon
-          aria-label={`EquipmentOption_${
-            isSelected ? 'checkbox-marked' : 'checkbox-blank-outline'
-          }`}
-          color={isSelected ? COLORS.blueEnabled : COLORS.darkGreyEnabled}
-          size="1.5rem"
-          name={isSelected ? 'checkbox-marked' : 'checkbox-blank-outline'}
-        />
-      ) : null}
+    <React.Fragment>
```

Comment:
You only need to write out a react fragment like this if you specify a key, otherwise it can be simplified to the shorthand react fragment:

```suggestion
    <>
```